index.html

```diff
@@ -0,0 +1,89 @@
+        alt="Helping Poor Farmer Grow Crops" />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug.
+        </figcaption>
+      </figure>
+      <section id="tribute-info">
+        <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
+        <ul>
+          <li><b>1914</b> - Born in Cresco, Iowa</li>
+          <li>
+            <b>1933</b> - Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as the
+            "National Youth Administration"
+          </li>
+          <li>
+            <b>1935</b> - Has to stop school and save up more money. Works
+            in the Civilian Conservation Corps, helping starving Americans. "I saw
+            how food changed them", he said. "All of this left scars on me."
+          </li>
+          <li>
+            <b>1937</b> - Finishes university and takes a job in the US
+            Forestry Service
+          </li>
+          <li>
+            <b>1938</b> - Marries wife of 69 years Margret Gibson. Gets
+            laid off due to budget cuts. Inspired by Elvin Charles Stakman, he
+            returns to school study under Stakman, who teaches him about breeding
+            pest-resistent plants.
+          </li>
+         
+          <li>
+            <b>1942</b> - Receives a Ph.D. in Genetics and Plant Pathology
+          </li>
+        
+          <li>
+            <b>1945</b> - Discovers a way to grown wheat twice each
+            season, doubling wheat yields
+          </li>
+          <li>
+            <b>1953</b> - crosses a short, sturdy dwarf breed of wheat
+            with a high-yeidling American breed, creating a strain that responds
+            well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+          </li>
+          <li>
+            <b>1962</b> - Visits Delhi and brings his high-yielding
+            strains of wheat to the Indian subcontinent in time to help mitigate
+            mass starvation due to a rapidly expanding population
+          </li>
+         
+          <li>
+            <b>1983</b> - helps seven African countries dramatically
+            increase their maize and sorghum yields
+          </li>
+          <li>
+            <b>2005</b> - states "we will have to double the world food
+            supply by 2050." Argues that genetically modified crops are the only way
+            we can meet the demand, as we run out of arable land. Says that GM crops
+            are not inherently dangerous because "we've been genetically modifying
+            plants and animals for a long time. Long before we called it science,
+            people were selecting the best breeds."
+          </li>
+        
+        </ul>
+    <h3>
+      More About Dr. Norman Borlaug
+      <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Click Me</a>
+    </h3>
```

Comment:
Please put it outside of the ```<section>``` and add it as a ```<footer>```. Then the HTML5 semantic will standards will meet.

Reply:
Great job being polite and giving a reason for the requested change.